File: ocaml/med_com_rubik.ml
(* https://www.codingame.com/training/medium/rubik%C2%AE *)

let n = int_of_string (input_line stdin) in
let ans = if (n == 1) then 1 else (6 * n * (n - 2) + 8) in
print_endline (string_of_int ans);
(* Write an action using print_endline *)
(* To debug: prerr_endline "Debug message"; *)

File: scala/med_com_rubik.scala
// https://www.codingame.com/training/medium/rubik%C2%AE

import scala.util._
import scala.io.StdIn._

object Solution extends App {
    val N = readInt
    var ans = 0
    if (N > 1) then {
        ans = 6 * N * (N - 2) + 8;
    } else {
        ans = 1;
    }
    println(ans)
}
// To debug: Console.err.println("Debug messages...")
